docs/guided-remediation.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Guided Remediation
+permalink: /experimental/guided-remediation/
+parent: Experimental Features
+nav_order: 3
+---
+
+# Guided Remediation
+
+Experimental
+{: .label }
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+The `osv-scanner fix` subcommand leverages [deps.dev](https://deps.dev) to provide automated and guided remediation of vulnerabilities in your project's dependencies by suggesting upgrades to dependencies.
+
+Currently, only npm `package.json` manifests and `package-lock.json` lockfiles are supported.
+
+{: .note }
+This feature is experimental and might change or be removed with only a minor version update.
+
+## Basic usage: interactive mode
+
+Interactive mode provides a step-by-step process to understand and fix vulnerabilities in your project.
+
+To run it, you can use the following command:
+
+```bash
+osv-scanner fix -M path/to/package.json -L path/to/package-lock.json
+```
+
+The command will launch the Guided Remediation TUI and begin scanning your manifest and lockfile for vulnerabilities. Use the arrow keys to navigate menu items and the `ENTER` key to select. `TAB` can be used to switch to and from the information panel of each option.
+
+![Screenshot of the initial interactive results screen, showing found vulnerabilities and the potential remediation approaches](images/guided-remediation-result.png)
+
+{: .highlight }
+
+> The initial screen allows you to change the vulnerability selection criteria. Each correspond to a [vulnerability selection flag](#vulnerability-selection):
+>
+> - Max dependency depth: `--max-depth`
+> - Min CVSS score: `--min-severity`
+> - Include dev only: `--ignore-dev`
```

Comment:
Surfacing @laserlemon's [comment](https://github.com/michaelkedar/osv-scanner/commit/3def3fad6b4baf7291fd81def72794d307e0a579#r139160741):
> First, thank you for writing this up! 💛
> 
> Is this option really function to "include dev only"? It seems that `--ignore-dev` would mean to only consider _runtime_ dependencies.

You're right that these are inverted of each other - setting `--ignore-dev` would result in `Include dev only: NO`.

I think I'm going to change the interactive screen to 'Exclude dev only' to make them consistent with eachother.